### ISSUE TITLE
Platforms organization

### ISSUE THREAD

**Issue by Tirifto** [pwnsdx/Tox-Website#23]
At the bottom of the website (at least according to the preview), there are five platforms listed: Windows 32-bit, Windows 64-bit, OS X, Android and "Others". Perhaps this was decided by popularity, but I'd say the platforms should be reorganized and presented in a better way.

![toxissue](https://cloud.githubusercontent.com/assets/10896954/6996944/81d2e792-dba6-11e4-8ee2-0b4a169e342e.png)

What bugs me is that Windows get listed twice, they're followed by OS X, OS X is followed by Android, which, unlike previous three entries, is a mobile platform, and the chain ends with Tux and "Others".

Tox supports many platforms, so putting "Others" button for the relatively unknown ones is understandable. I see that iOS client may not be ready for average user yet, so it's also understandable for it to be there, but I would **definitely** not shove GNU/Linux into background. It is the only platform where users are able to have true privacy, security and freedom. Other listed platforms oppress these same values Tox tries to support, so I don't think they should be advertised more.

**TL;DR**, I suggest: Reducing Windows to one button which opens up (or links to) further selection between 32-bit and 64-bit, and addition of GNU/Linux button.

**Comment by pwnsdx** [comment]
I didn't done that and I'm totally agree with you.
